src/config/kernel.md

```diff
@@ -76,15 +90,27 @@ omit_drivers+=" radeon "
 ```
 
 Now initramfs needs to be regenerated to make the changes take effect on the
-next reboot.
+next reboot:
 
 ```
 # dracut --force
 ```
 
 #### mkinitcpio
 
-XXX: add example of blacklisting kernel modules for mkinitcpio
+To blacklist modules from being included in a mkinitcpio initramfs a `.conf`
+file needs to be created like `/etc/modprobe.d/radeon.conf` with the contents:
+
+```
+blacklist radeon
```

Comment:
This is just literally a copy of 2 headings ago. Could we just compromise and say something like:

To blacklist modules in a mkinitcpio initramfs, create a `.conf` file in `/etc/modprobe.d/` as above, then regenerate the image:

```
# mkinitcpio -p linux
```

The changes will take effect on the next reboot.